pkg/client/src/app/pages/applications/manage-imports/manage-imports.tsx

```diff
@@ -269,6 +267,7 @@ export const ManageImports: React.FC = () => {
   const deleteRow = (row: ApplicationImportSummary) => {
     dispatch(
       confirmDialogActions.openDialog({
+        // t("terms.tag(s)")
```

Comment:
Are we using this?

Reply:
Yest this is work around otherwise i18n is going to hit us with translations.json files.